ci-operator/jobs/infra-periodics.yaml

```diff
@@ -2616,3 +2616,59 @@ periodics:
       resources:
         requests:
           cpu: 500m
+- agent: kubernetes
+  cluster: app.ci
+  decorate: true
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: release
+  cron: '@daily'
+  labels:
+    ci.openshift.io/role: infra
+  name: periodic-trigger-managed-clonerefs-mabc
+  spec:
+    containers:
+    - command:
+      - /bin/bash
+      args:
+      - -c
+      - |
+        set -o errexit
+        set -o nounset
+        set -o pipefail
+
+        export KUBECONFIG='/etc/build-farm-credentials/kubeconfig'
+
+        curl -sL 'https://github.com/jqlang/jq/releases/download/jq-1.7/jq-linux-amd64' >/tmp/jq
```

Comment:
We shouldn't download the `jq` each time the job runs. Instead, we need to maintain a specific image and move the rest of the logic to a script.